components/org.wso2.carbon.status.dashboard.web/src/siddhi-app/ComponentHistory.jsx

```diff
@@ -352,43 +389,47 @@ export default class ComponentHistory extends React.Component {
                 <Redirect to={{pathname: `${window.contextPath}/logout`}}/>
             );
         }
-        return (
-            <div style={{backgroundColor: '#222222'}}>
-                <Header/>
-                <div className="navigation-bar">
-                    <Link to={window.contextPath}><FlatButton label="Overview >"
-                                                              icon={<HomeButton color="black"/>}/></Link>
-                    <Link to={window.contextPath + '/worker/' + this.props.match.params.id }>
-                        <FlatButton label={this.state.workerID + " >"}/></Link>
-                    <Link
-                        to={window.contextPath + '/worker/' + this.props.match.params.id + '/siddhi-apps/' +
-                        this.props.match.params.appName + "/" + this.state.statsEnable}>
-                        <FlatButton label={this.props.match.params.appName + " >"}/>
-                    </Link>
-                    <RaisedButton label={this.props.match.params.componentId} disabled disabledLabelColor='white'
-                                  disabledBackgroundColor='#f17b31'/>
-                </div>
-                <div className="worker-h1">
-                    <h2 style={{marginLeft: 40}}> {this.props.match.params.componentId} Metrics </h2>
+        if(this.state.hasViewerPermission) {
```

Comment:
Check whther proper whitespaces are used.. (formatting issues)

Reply:
Fixed in 50c0b3ff27410966326b69c3e721733fe60b591b